examples/favorites_advanced/getx_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart

```diff
@@ -11,7 +11,7 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
   final controller = Get.find<PuppyListController>();
 
   AppBarTextField _searchAppBar() => AppBarTextField(
-        title: const Text('Search for Puppies'),
+        title: _createTitle(),
```

Comment:
@RomanovaPrime, the convention for building widgets is _build{NameOfTheWidget}. In your case it could be _buildTitle()

Reply:
Renamed.